### ISSUE TITLE
pip install-ed rapidwright dies on Java exception

### ISSUE THREAD

**Issue by eddieh-xlnx** [Xilinx/RapidWright#259]
```
bash-4.2$ python3
Python 3.6.8 (default, Nov 16 2020, 16:55:22) 
[GCC 4.8.5 20150623 (Red Hat 4.8.5-44)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import rapidwright
f>>> from com.xilinx.rapidwright.design import Design
>>> d = Design.readCheckpoint('does_not_exist.dcp')
==============================================================================
==                            Reading DCP: a.dcp                            ==
==============================================================================
java.io.FileNotFoundException: does_not_exist.dcp (No such file or directory)
	at java.util.zip.ZipFile.open(Native Method)
	at java.util.zip.ZipFile.<init>(ZipFile.java:225)
	at java.util.zip.ZipFile.<init>(ZipFile.java:155)
	at java.util.zip.ZipFile.<init>(ZipFile.java:169)
	at b.a(Unknown Source)
	at com.xilinx.rapidwright.design.Design.readCheckpoint(Unknown Source)
Exiting due to read checkpoint failure...
bash-4.2$
```

Ideally, the Java exception should be converted into a Python exception

**Comment by clavin-xlnx** [comment]
Yes, there are some vestigial calls buried deep that invoke `System.exit()` (from `MessageGenerator.briefErrorAndExit()`) which are not desirable.  I believe that is what is happening here.  I will try to purge them in the next release.

**Comment by eddieh-xlnx** [comment]
Fixed by #260 and tested in #285.
